docs/operations/metrics.md

```diff
@@ -244,7 +251,7 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`segment/loadQueue/count`|Number of segments to load.|server.|Varies.|
 |`segment/dropQueue/count`|Number of segments to drop.|server.|Varies.|
 |`segment/size`|Total size of used segments in a data source. Emitted only for data sources to which at least one used segment belongs.|dataSource.|Varies.|
-|`segment/count`|Number of used segments belonging to a data source. Emitted only for data sources to which at least one used segment belongs.|dataSource.|< max|
+|`segsegment/count`|Number of used segments belonging to a data source. Emitted only for data sources to which at least one used segment belongs.|dataSource.|< max|
```

Comment:
Is this a typo?
Reading the description, it seems that this should rather be reported as `segment/used/count`. What do you think?

Reply:
yes a typo.

`segment/used/count` sounds like a better name. Unfortunately, it's an existing metric that people might rely on, so if we change the name, I think we should consider how to minimize disruption for users on an upgrade. I don't know of a good pattern for this, so I'm going to skip making that change